src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -47,6 +50,9 @@ def load_arguments(self, _):
         c.argument('proxy_cert', options_list=['--proxy-cert'], arg_group='Proxy', type=file_type, completer=FilesCompleter(), help='Path to the certificate file for proxy')
         c.argument('disable_proxy', options_list=['--disable-proxy'], arg_group='Proxy', action='store_true', help='Disables proxy settings for agents')
         c.argument('auto_upgrade', options_list=['--auto-upgrade'], help='Flag to enable/disable auto upgrade of arc agents. By default, auto upgrade of agents is enabled.', arg_type=get_enum_type(["true", "false"]))
+        c.argument('container_registry_repository', options_list=['--registry-repository'], help='Custom container registry repository.')
```

Comment:
Same comments apply here.(as given for connectedk8s connect command params)

Reply:
ack